validateIntrospectorScript.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+found_error=0
+
+INTROSPECT_DOMAIN_SCRIPT_FILE=$(find "$(pwd -P)" -type f -name "introspectDomain.sh" | tail -n 1)
```

Comment:
Instead of 'find' can we hard code the exact file location? With ^^^ more than one occurrence of introspectDomain.sh could throw off the check...

Reply:
I have check to look for only the last occurrence (`tail -n 1`).